Gemfile.lock

```diff
@@ -291,7 +291,7 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 RUBY VERSION
-   ruby 2.6.2p47
+   ruby 2.7.0p0
 
 BUNDLED WITH
-   1.17.2
+   2.1.2
```

Comment:
CI will be failed with Ruby 2.7.0 because bundler 1.17.2 depends on older version of Ruby. So we had to update the version of bundler.

Reply:
ok